app/lib/search/search_service.dart

```diff
@@ -383,7 +384,7 @@ class SearchQuery {
         order == null &&
         parsedQuery.packagePrefix == null &&
         (platform == null || platform.isEmpty) &&
-        (tagsPredicate == null || tagsPredicate._values.isEmpty);
+        (tagsPredicate == null || tagsPredicate.isEmpty);
     if (isEmpty) return false;
```

Comment:
Why is an empty search query not valid?

Reply:
I remember that at one point it was used to decide between regular pagination and search requests, but apparently that code is gone now, and this method is only used in tests. I'll do a follow-up cleanup once this PR lands.